### PR TITLE
Set MediaInfo defaults

### LINKED ISSUE #2598: Set MediaInfo=true by default
The default value in RendererConfiguration should be set to `true`, but without changing behaviour for any existing renderers, to avoid breaking compatibility for them.
So existing renderers that do not have that line, or have set it to false, should still have it as false, while those who set it to `true` can have it removed.

The main benefit of this change is that new renderers going forward will not have to have the line at all, because it is much better to use MediaInfo than not.

### PR DESCRIPTION
**Pull request by ahmedadan**
- Any file that had explicitly set false was left as false as well as any file that did not have MediaInfo set was explicitly set to false.
- Changed Default Behaviour in `RendererConfiguration.java`

Closes #2598

### CODE REVIEW

File: src/main/external-resources/renderers/Miracast-M806.conf [hunk 1]
```diff
@@ -13,7 +13,7 @@ RendererIcon = miracast.png
 # {friendlyName=LOLLIPOP-1C3A18-DMR, manufacturer=Rockchip, modelName=Rockchip Media Renderer, modelNumber=1.0, modelDescription=Rockchip Media Renderer, DLNA(DMR), manufacturerURL=http://www.rock-chips.com, modelURL=http://www.rock-chips.com}
 # ============================================================================
 
-#MediaInfo = true
+MediaInfo = false
```
Comment (ahmedadan) [inline src/main/external-resources/renderers/Miracast-M806.conf:16]: This was commented out so I set it to false, I can change that if we think true would be good in this situation.

**Comment by SubJunk** [comment]
@ahmedadan thank you for this, I will review later today :)

**Review by SubJunk — CHANGES_REQUESTED** [review]
The default still needs to be changed in `RendererConfiguration.java`

File: src/main/java/net/pms/configuration/RendererConfiguration.java [hunk 2]
```diff
@@ -2119,7 +2119,7 @@ public String toString() {
 	 * @return whether to use MediaInfo
 	 */
 	public boolean isUseMediaInfo() {
-		return getBoolean(MEDIAPARSERV2, false) && LibMediaInfoParser.isValid();
+		return getBoolean(MEDIAPARSERV2, true) && LibMediaInfoParser.isValid();
```
Comment (ahmedadan) [inline src/main/java/net/pms/configuration/RendererConfiguration.java:2122]: After reading the chained constructors I found what I believe to be the default setting, let me know if I misunderstood or made a bad change @SubJunk @valib

File: src/main/external-resources/renderers/DirecTV.conf [hunk 3]
```diff
@@ -21,3 +21,4 @@ CustomFFmpegOptions = -ac 2 -c:a mp2 -ab 192k
 CustomMencoderOptions = -channels 2 -lavcopts acodec=mp2:abitrate=192
 KeepAspectRatio = true
 HalveBitrate = true
+MediaInfo = true
```
Comment (valib) [inline src/main/external-resources/renderers/DirecTV.conf:24]: All `MediaInfo = true` should be removed from all renderers.conf

File: src/main/external-resources/renderers/Google-Android.conf [hunk 4]
```diff
@@ -14,4 +14,4 @@ MuxDTSToMpeg = true
 H264Level41Limited = false
 StreamExtensions = mkv,hdmov,hdm,flac,fla,dts,ogg,asf,asx,m2v,mp4,mpg,mpeg,avi,mov,wmv
 ChunkedTransfer = true
-HalveBitrate = true
+HalveBitrate = True
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Google-Android.conf:17]: Can you revert this change please?

File: src/main/external-resources/renderers/Samsung-BDC6800.conf [hunk 5]
```diff
@@ -32,3 +32,4 @@ Supported = f:wmv      v:vc1|wmv            a:wma                    m:video/x-m
 # Supported audio formats:
 Supported = f:mp3   m:audio/mpeg
 Supported = f:wma   m:audio/x-ms-wma
+MediaInfo = false
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Samsung-BDC6800.conf:35]: This is interesting since I think we have found a bug with this configuration - it has the `Supported` lines without the `MediaInfo = true`. In this case let's leave it, so can you please remove this line?

**Review by SubJunk — CHANGES_REQUESTED** [review]
Nice work, just a couple of minor changes to make. Can you also sync with master after making the changes?

**Comment by SubJunk** [comment]
If you sync with master I hope it will pass now

**Comment by ahmedadan** [comment]
> 
> 
> If you sync with master I hope it will pass now

Looks like you fixed it, Let me know if you want me to squash into a single commit but otherwise it's all done on my end

**Comment by SubJunk** [comment]
@ahmedadan nice job, no need to squash, I will squash when it merges. I'm just going to test it tonight before merging, in case there are any unanticipated problems

**Comment by SubJunk** [comment]
This is working well for me so I'll merge

**Review by SubJunk — APPROVED** [review]
